cluster/vagrant/setup_kubernetes_master.sh

```diff
@@ -50,6 +50,31 @@ else
   kubectl create -f kube-$NETWORK_PROVIDER.yaml
 fi
 
+{
+  pushd /vagrant
+  source hack/config.sh
+  popd
+
+  for SVC in spice-proxy:3128 virt-controller:8182 virt-api:8182 haproxy:8184 virt-manifest:8186;
```

Comment:
Do we really need a virt-controller service endpoint?  It seems like we could just remove both the external and internal virt-controller endpoints.

Reply:
I just keep compatibility with the current state. Happy to drop all unnecessary mappings in a separate patch.